src/Catalyst.Core.Modules.Kvm/KvmModule.cs

```diff
@@ -22,23 +22,29 @@
 #endregion
 
 using Autofac;
+using Autofac.Core;
 using Catalyst.Abstractions.Kvm;
 using Nethermind.Core.Specs;
 using Nethermind.Evm;
 using Nethermind.Logging;
 using Nethermind.Store;
+using Module = Autofac.Module;
 
 namespace Catalyst.Core.Modules.Kvm
 {
     public class KvmModule : Module
     {
+        public const string DeltaBuilderComponentsName = "BuilderState";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Change this constant to a 'static' read-only property.](https://app.codacy.com/app/catalyst-network/Catalyst.Framework/pullRequest?prid=4585091)